adapters/platform_linux.c

```diff
@@ -91,5 +96,7 @@ void platform_deinit(void)
 #endif /* DONT_USE_UPLOADTOBLOB */
 #ifdef USE_OPENSSL
     tlsio_openssl_deinit();
+#elif USE_WOLFSSL
+    tlsio_wolfssl_deinit();
 #endif
```

Comment:
>endif [](start = 1, length = 5)

windows platform also requires this?
[https://github.com/Azure/azure-c-shared-utility/blob/51555845f446a7f0be4e3442a321c558af04910b/adapters/platform_win32.c#L235](https://github.com/Azure/azure-c-shared-utility/blob/51555845f446a7f0be4e3442a321c558af04910b/adapters/platform_win32.c#L235)

Reply:
On Windows only Schannel is recommended and officially supported as is Secure Transport on OSX. 
I won't add these to Windows.